server/schemas/typeDefs.js

```diff
@@ -1,20 +1,49 @@
 const typeDefs = `
-  type Profile {
+  type User {
     _id: ID
-    name: String
-    skills: [String]!
+    username: String
+    password: String
+    buddy: ID
+    availability: Boolean
+    role: String
+    expertise: String
+    conversations: [Conversation]!
```

Comment:
Does User model need to be changed to reflect this?

Reply:
hmm...I don't think so as this was created based off the User model. Which part are you looking to modified?

Reply:
hm...I think the current setup should work. When there's multiple conversation associated with the user, we will just have to grab the specific conversation ID, which line#39 in typeDefs should query for that?

Reply:
@Jasmineleeyt sounds good, we shouldn't have that happen anyways but lgtm!

Reply:
Thank you for your review!